### PR TITLE
PRO-1002 Re-Default Token list for new users

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
https://linear.app/pillarproject/issue/PRO-1002/pillar-wallet-default-token-list-for-new-users

## Description

- Manage default tokens list locally
- Show default list in particular order
- Show default list for new user (offline/online)

## Motivation and Context

- List manage locally
- https://linear.app/pillarproject/issue/PRO-1002#comment-fb68fe0a
- https://linear.app/pillarproject/issue/PRO-1002#comment-341fd04e

## How Has This Been Tested?

- Tested locally on Samsung A50 (online/offline)
- Tested locally on Simulator (online/offline)
- 

## Screenshots (if appropriate):
Offline:
https://user-images.githubusercontent.com/78720269/226817311-9fb6aede-70a9-4b48-b10b-4509525d0477.mp4

Online
https://user-images.githubusercontent.com/78720269/226826076-8a8bc57d-ba1a-442f-b91f-761d87ad4a48.mp4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/actions/etherspotActions.js [hunk 1]
```diff
@@ -283,65 +279,21 @@ export const fetchDefaultTokens = () => {
...
-    const tokens: any[] = [];
-    await Promise.all(
-      chainTokens?.map(async (item: any) => {
-        if (!isEmpty(item)) {
-          tokens.push(...item);
-          await getExchangeRates(item[0].chain, item, async (rates) => {
-            if (rates) await dispatch(updateRatesAction(item[0].chain, rates));
-          });
-        }
+        await getExchangeRates(chain, chainAssets, async (rates) => {
```
Comment (IAmKio) [inline src/actions/etherspotActions.js:288]: What happens if one of these rates fail?

File: src/actions/etherspotActions.js [hunk 2]
```diff
@@ -283,65 +279,21 @@ export const fetchDefaultTokens = () => {
...
+          if (rates) await dispatch(updateRatesAction(chain, rates));
```
Comment (IAmKio) [inline src/actions/etherspotActions.js:289]: Do we need to `await` this `dispatch`?

File: src/assets/tokens/BUSD-BD1.png
Comment (IAmKio) [inline src/assets/tokens/BUSD-BD1.png:1]: The quality of this is too poor

File: src/assets/tokens/avax.png
Comment (IAmKio) [inline src/assets/tokens/avax.png:1]: The quality is a bit too low for this

File: src/assets/tokens/fxs.png
Comment (IAmKio) [inline src/assets/tokens/fxs.png:1]: The quality is a bit too low for this

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few changes to the asset icons, some of them are too low

File: src/actions/etherspotActions.js (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/actions/etherspotActions.js:288]: Call catch fn

```
} catch (error) {
          reportErrorLog('Fetch Rates failed: request error', {
            error,
            assetsContractAddresses,
          });
        }
```

File: src/actions/etherspotActions.js (hunk 2 above)
Comment (hemadri-tank-bacancy) [reply src/actions/etherspotActions.js:289]: not mandatory, I removed it

File: src/assets/tokens/BUSD-BD1.png
Comment (hemadri-tank-bacancy) [reply src/assets/tokens/BUSD-BD1.png:1]: I did update it

File: src/assets/tokens/avax.png
Comment (hemadri-tank-bacancy) [reply src/assets/tokens/avax.png:1]: I did update it

File: src/assets/tokens/fxs.png
Comment (hemadri-tank-bacancy) [reply src/assets/tokens/fxs.png:1]: I did update it

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**